src/jobs/jobs.service.ts

```
@@ -67,208 +69,95 @@ export class JobsService {
     id: number,
-    updateApplicationDto: UpdateApplicationDto,
+    dto: UpdateApplicationDto,
   ): Promise<Application> {
     const application = await this.findApplicationById(id);
-    Object.assign(application, updateApplicationDto);
+    Object.assign(application, dto);
     return this.applicationRepository.save(application);
   }
 
+  async findApplicationById(id: number): Promise<Application> {
+    const app = await this.applicationRepository.findOne({ where: { id: id.toString() } });
```

ESLint

[error] Replace `·where:·{·id:·id.toString()·}` with `⏎······where:·{·id:·id.toString()·},⏎···`

(prettier/prettier)